### DISCUSSION TITLE
Intellectual property and licensing

### DISCUSSION THREAD

**Discussion by benlangfeld** [wmo-im/iwxxm#243] [Q&A] [answered]
It is not clear from reading the content of this repository to whom the copyright in its contents belongs and under which license terms it is made available to the public. Can anyone clarify where this information might be found?

**Comment by blchoy** [comment, marked as answer]
I am not expert on this matter.  May be @amilan17 could help?

**Reply by blchoy** [reply]
Found a [WMO webpage](https://public.wmo.int/en/copyright) on this matter.  I trust this should also apply to the contents under https://github.com/wmo-im.
